<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d_ask</class>
 <widget class="QDialog" name="d_ask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#d_ask {
    border-image: url(:/bg/resource/bg_most_3.png);
}

#lineEdit_ask_title{
    background-color: white;
    border: 2px solid #868282;
    border-radius: 5px;
}

#lineEdit_ask_title:hover {
    background-color: #F5EFEF;
    border: 2px solid #868282;
    border-radius: 5px;
}

#textEdit_ask_text{
	border: none;
    background: #F5F1F1
}

#pushButton_ask_ok{
    border-image: url(:/icons/resource/ok.png);
}

#pushButton_ask_ok:hover{
    border-image: url(:/icons/resource/ok_h.png);
}

#pushButton_ask_return{
    border-image: url(:/icons/resource/back.png);
}

#pushButton_ask_return:hover{
    border-image: url(:/icons/resource/back_h.png);
}</string>
  </property>
  <widget class="QTextEdit" name="textEdit_ask_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>431</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'等线'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ask_return">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>3</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ask_ok">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>660</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ask_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
